docs/examples/tour/basic.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <el-button type="primary" @click="open = true">Begin Tour</el-button>
+
+  <el-divider />
+
+  <el-space>
+    <el-button ref="ref1">upload</el-button>
+    <el-button ref="ref2" type="primary">save</el-button>
+    <el-button ref="ref3" :icon="MoreFilled" />
+  </el-space>
+
+  <el-tour v-model="open">
+    <el-tour-step
+      :target="ref1?.$el"
+      title="Upload File"
+      description="Put you files here."
+    >
+      <template #cover>
+        <img
+          src="https://user-images.githubusercontent.com/5378891/197385811-55df8480-7ff4-44bd-9d43-a7dade598d70.png"
+          alt="tour.png"
+        />
+      </template>
+    </el-tour-step>
+    <el-tour-step
+      :target="ref2?.$el"
+      title="Save"
+      description="Save your changes"
+    />
+    <el-tour-step
+      :target="ref3?.$el"
+      title="Other Actions"
+      description="Click to see other"
+    />
+  </el-tour>
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue'
+import { type ElButton } from 'element-plus'
```

Comment:
## Suggestion

```jsx
import type { ElButton } from 'element-plus'
```

Reply:
I wrote this at first, but eslint reported an error.
<img width="863" alt="image" src="https://github.com/element-plus/element-plus/assets/27912232/b4bfbd73-0f06-4589-ad92-ae8d70a25a21">

Reply:
Should I add eslint disable comments, or modify the .eslintrc.json file